patches/net/minecraft/client/multiplayer/MultiPlayerGameMode.java.patch

```diff
@@ -1,16 +1,26 @@
 --- a/net/minecraft/client/multiplayer/MultiPlayerGameMode.java
 +++ b/net/minecraft/client/multiplayer/MultiPlayerGameMode.java
-@@ -120,9 +_,8 @@
+@@ -120,14 +_,15 @@
                  } else if (blockstate.isAir()) {
                      return false;
                  } else {
 -                    block.playerWillDestroy(level, p_105268_, blockstate, this.minecraft.player);
                      FluidState fluidstate = level.getFluidState(p_105268_);
 -                    boolean flag = level.setBlock(p_105268_, fluidstate.createLegacyBlock(), 11);
-+                    boolean flag = blockstate.onDestroyedByPlayer(level, p_105268_, minecraft.player, false, fluidstate);
-                     if (flag) {
-                         block.destroy(level, p_105268_, blockstate);
+-                    if (flag) {
+-                        block.destroy(level, p_105268_, blockstate);
++                    BlockState removedBlockState = block.playerWillDestroy(level, p_105268_, blockstate, minecraft.player);
++                    boolean removed = blockstate.onDestroyedByPlayer(level, p_105268_, minecraft.player, false, fluidstate);
```

Comment:
I believe if we keep this boolean named `flag` then we don't have to be patching the two `return` statements?